amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -2647,10 +2653,9 @@ type Expenditure @model {
   finalizedAt: AWSTimestamp
   hasReclaimedStake: Boolean
   """
-  The motions created to fund this expenditure, if there are any
+  The motions created that relate to this expenditure, if there are any
   """
-  fundingMotions: [ColonyMotion]
-    @hasMany(indexName: "byExpenditureId", fields: ["id"])
+  motions: [ColonyMotion] @hasMany(indexName: "byExpenditureId", fields: ["id"])
```

Comment:
Since we'll have many kinds of motions, I suggest merging into one field and filtering by action type whenever it's necessary to select a certain type